### ISSUE TITLE
show map with  local mbtiles(MaplyMBTileSource).

### ISSUE THREAD

**Issue by duzvik** [mousebird/WhirlyGlobe#851]
Hello, trying to show  map with local mbtiles (raw JPEG in tile_data field) but got black screen.

Here is code in viewDidLoad:
 
```swift 
        mapView = MaplyViewController(mapType: .typeFlat)
        mapView!.clearColor = UIColor.white
        view.addSubview(mapView!.view)
        mapView!.view.frame = view.bounds
        addChildViewController(mapView!)
            
        if let tileSource = MaplyMBTileSource(mbTiles: "test_tiles"),
            let layer = MaplyQuadImageTilesLayer(coordSystem: tileSource.coordSys, tileSource: tileSource) {
            layer.handleEdges = true
            layer.coverPoles = true
            layer.requireElev = false
            layer.waitLoad = false
            layer.drawPriority = 0
            layer.singleLevelLoading = false
            mapView!.add(layer)
        }
        mapView?.height = 1
        let position = MaplyCoordinateMakeWithDegrees( 37.8028, 48.0159)
        mapView?.animate(toPosition: position, time: 1)
```

**Comment by mousebird** [comment]
Try it with the GeographyClass example and make sure you see something with this setup.

**Comment by duzvik** [comment]
Just updated GeographyClassTestCase.m with my tiles and my coordinates. Same: black screen.
I've tested my tiles with MapTiller, and tiles are OK(successfully showing needed area)
Also I'm 100% sure that coordinates are OK - i've used them geography-class_medres tiles and mapView points to needed  place. 
And of course I've tried different zoom levels. (my tiles, according to metadata table, has 4-14 zoom level)
